libs/utils/src/hash_map_private.h

```diff
@@ -29,13 +29,17 @@
 
 #include "hash_map.h"
 
-unsigned int hashMap_hashCode(const void* toHash);
-int hashMap_equals(const void* toCompare, const void* compare);
+#ifdef __cplusplus
+extern "C" {
+#endif
 
-void hashMap_resize(hash_map_pt map, int newCapacity);
-hash_map_entry_pt hashMap_removeEntryForKey(hash_map_pt map, const void* key);
-hash_map_entry_pt hashMap_removeMapping(hash_map_pt map, hash_map_entry_pt entry);
-void hashMap_addEntry(hash_map_pt map, int hash, void* key, void* value, int bucketIndex);
+CELIX_UTILS_DEPRECATED_EXPORT unsigned int hashMap_hashCode(const void* toHash);
```

Comment:
Why export? `CELIX_UTILS_DEPRECATED` seems more reasonable.

Reply:
The export macros are indeed not needed, removing them. Deprecated is correct but IMO mainly valuable on a public api.